Gopkg.toml

```diff
@@ -68,10 +68,18 @@
   name = "github.com/json-iterator/go"
   version = "1.1.4"
 
+[[override]]
+  name = "k8s.io/client-go"
+  branch = "master"
+
 [[override]]
   name = "gopkg.in/square/go-jose.v2"
   version = "2.1.7"
 
+[[override]]
+  source = "https://github.com/fsnotify/fsnotify/archive/v1.4.7.tar.gz"
```

Comment:
is this actually needed?

Reply:
when using dep ensure, I was getting a `panic: version queue is empty` error, and adding this solved it. Reference: https://github.com/golang/dep/issues/1799